examples/two_mass_spring_damper/run.py

```diff
@@ -0,0 +1,94 @@
+import matplotlib.pyplot as plt
+from numpy import linspace
+from pydy.system import System
+from sympy import symbols, Matrix
+from sympy.physics.mechanics import dynamicsymbols, eombase
+
+# Define the dynamic symbols
+
+x1, x2, u1, u2 = dynamicsymbols('x1 x2 u1 u2')
+
+# Define the constant symbols
+
+m1, c1, k1 = symbols('m1 c1 k1')
+m2, c2, k2 = symbols('m2 c2 k2')
+
+# Define the mass matrix and forcing vector
+
+mm = Matrix([[m1,  0],
+             [0,  m2]])
+f = Matrix([-(k1+k2)*x1 + k2*x2 - (c1+c2)*u1 + c2*u2,
+            k2*x1 - k2*x2 + c2*u1 - c2*u2])
+
+mm_full = Matrix([[1, 0,  0,  0],
+                  [0, 1,  0,  0],
+                  [0, 0, m1,  0],
+                  [0, 0,  0, m2]])
+f_full = Matrix([u1,
+                 u2,
+                 -(k1+k2)*x1 + k2*x2 - (c1+c2)*u1 + c2*u2,
+                 k2*x1 - k2*x2 + c2*u1 - c2*u2])
+
+# Form the rhs of q' = G(q, u, t, r, p). Kinematic equation
+
+G = Matrix([u1, u2])
+
+# Form the rhs of the dynamics equations
+
+RHS = mm_full.inv()*f_full
+
+# Form the remaining input lists for the equations of motion class
+
+coordinates = (x1, x2)
+speeds = (u1, u2)
+states = (x1, x2, u1, u2)
+
+# Form the kinetic energy to attach to one of the EOM instances as an output
+# equation
+
+KE = 0.5 * (m1*u1**2 + m2*u2**2)
+out_eqns = {"kinetic_energy": KE}
```

Comment:
Most things are passed into the `*Method`s classes as:

```python
exprs = [expr1, expr2, expr3, ...]
```

Where `expri` represents the left hand side of:

```
expri = 0
```

In the `System` class we use mostly dictionaries to map sympy symbolics to numerics (floats or arrays of floats). `System` was designed years after `*Method`s and has some nice advantages. So in `System` you could do something like:

```python
sys.plot_state(q1, q2, q3, u1)
```

Where q1, q2, q3, u1 are SymPy symbols. They can be easily printed in the plot using LaTeX. But you may also sometimes what to know what a state is, i.e. q1 is the angle angle between the arm of the pendulum and the ground...

If you use symbols for the output equations you can do:

```python
sys.plot(q1, q2, ke)
```

Where you can know include other symbols that represent the output equations.

I don't know what the best idea for all these API choices are, keep in mind I'm just throwing out ideas and trying to insert some of the historical information about the API design.